src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1019,7 +1019,21 @@ int RPCCall::fromCommandLine (const std::vector<std::string>& vCmd)
         }
         else
         {
-            auto const setup = setup_ServerHandler(getConfig(), std::cerr);
+            ServerHandler::Setup setup;
+            try
+            {
+                std::stringstream ss;
+                setup = setup_ServerHandler(getConfig(), ss);
+            }
+            catch(...)
+            {
+                // eat this
```

Comment:
Figure out where the config file is located is a hard problem, and not one worth solving again for this temporary code that will go away when we move the command line client to another tool.